README.md

```diff
@@ -1,5 +1,7 @@
 # HR-TIL
 
+![til](https://cloud.githubusercontent.com/assets/11463275/26385291/8125cf72-400c-11e7-8779-1fdb802c9d6f.png)
```

Comment:
I guess I'd want to see the image in the repository itself, perhaps in the `app/assets/images` directory. So, you'd actually add the image to this pull request, on its own commit along with this markdown change. Then, we could link to it directly with something like `![til](https://raw.githubusercontent.com/hashrocket/hr-til/master/app/assets/images/banner.png)`. Does that make sense?